### PR TITLE
fix(ui): ui compile fail

### PR DESCRIPTION
**Pull request by WanQuanXie**
This problem is caused by the `ThemeSwitcher` component be refactored with not checked completely.

solution:
remove both of them, a new feature is coming soon -  mobile device support setting NavBar.

### CODE REVIEW

**Comment by nirabo** [comment]
> This problem is caused by the `ThemeSwitcher` component refactor with not checked completely.
> 
> solution: remove both of them, a new feature is coming soon - mobile device support setting NavBar.

Yes, this fixed the build problem. Thanks

File: ui/components/Navbar.tsx
Comment (malparty) [inline ui/components/Navbar.tsx:54]: Should we also remove the `import ThemeSwitcher from './theme/Switcher';` at line 5? Same for `ui/components/EmptyChat.tsx:2` :)

Comment (WanQuanXie) [reply ui/components/Navbar.tsx:54]: Although it doesn't matter much and will be auto removed by the compiler when on compiling stage, it's prefer to remove for code clean. I'll work on it no worries. Thanks for pointing it out

**Review by malparty — APPROVED** [review]

**Comment by meokey** [comment]
I experienced the same error. Look forward this fix to help. Thx

**Comment by namnx228** [comment]
Hi, 
I hope this PR can be merged soon, unless there is a particular reason against that. The latest release is broken.

**Comment by WanQuanXie** [comment]
> Hi, I hope this PR can be merged soon, unless there is a particular reason against that. The latest release is broken.

I want but I can't. I'm not the maintainer.

**Comment by huyn3195** [comment]
LGTM

**Comment by ItzCrazyKns** [comment]
Thanks!

**Pull request merged**
